src/legacy/core_plugins/interpreter/public/functions/font.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// @ts-ignore no @typed def
+import inlineStyle from 'inline-style';
+import { i18n } from '@kbn/i18n';
+import { openSans } from '../../common/lib/fonts';
+import { ExpressionFunction } from '../../types';
+import { CSSStyle, FontFamily, FontWeight, Style, TextAlignment, TEXT_ALIGNMENTS } from '../types';
```

Comment:
Question on the location (not sure if I know the right answer, just thinking aloud):

Since we know types for fonts are only going to be in `/public` (not `/server`), it makes sense that we would export them here. But would it make more sense to colocate them with the other types in `interpreter/types`, and then simply re-export them from `interpreter/public`?

It's the first time we've run into this situation, but I suppose it's a question of conventions. Do we want to centralize all types that we know are shared at a top-level (alongside `/server` and `/public`), and then export them as needed to either server or public or both? Or do we take items that we know are only ever one or the other and create another types directory inside of public?

Regardless I don't think this is important to solve now -- since this will all be moving to `plugins/data` eventually we can sort this out at that time. Just thought I'd raise the question to see if you had strong feelings on it.